### ISSUE TITLE
Provide templates for netpol scenarios

### ISSUE THREAD

**Issue by mattfenwick** [jayunit100/network-policy-subproject#41]
Instead of arbitrary targeting, focus on expected use cases:

Pod -> Pod
Pod -> Service
Pod -> Namespace

Service -> Pod
Service -> Service
Service -> Namespace

Namespace -> Pod
Namespace -> Service
Namespace -> Namespace

etc.

**Comment by jayunit100** [comment]
Yup, so this means we need a total of 6 different 'template' constructions, 

```
Pod -> Pod
Pod -> Service
Pod -> Namespace

Service -> Service
Service -> Namespace

Namespace -> Namespace
```
